packages/ai/src/components/ai-module/ai-module-header.tsx

```diff
@@ -69,14 +98,7 @@ export const AIModuleHeader = React.forwardRef<HTMLDivElement, AIModuleHeaderPro
         </>
       ) : (
         <div className="sk-ai-module-header-title">
-          <Avatar
-            className="sk-ai-module-header-avatar"
-            size={variant === 'alt' ? 'lg' : docked ? 'md' : 'sm'}
-            imageElement={typeof assistant.avatar !== 'string' ? assistant.avatar : undefined}
-            imageUrl={typeof assistant.avatar === 'string' ? assistant.avatar : undefined}
-            imageAlt=""
-            aria-hidden
-          />
+          {getAvatar()}
```

Comment:
About calling component functions instead of using <Avatar/> in the return. It makes them non-components and will hence not get their own life-cycles. So the main component won't be able to offload work to these sub-components, it will be one whole component life-cycle. 
https://react.dev/reference/rules/react-calls-components-and-hooks#never-call-component-functions-directly
Further in action example:
https://dev.to/igor_bykov/react-calling-functional-components-as-functions-1d3l